src/python/pants/init/logging.py

```diff
@@ -60,8 +60,8 @@ def formatException(self, exc_info):
         )
 
         return (
-            f"{stacktrace}\n\n({debug_instructions}See {doc_url('troubleshooting')} for common issues. "
-            f"Consider reaching out for help: {doc_url('getting-help')}.)"
+            f"{stacktrace}\n\n{debug_instructions}\nSee {doc_url('troubleshooting')} for common "
```

Comment:
The parens seem unnecessary, and breaking this into multiple lines makes it more readable